OCPBUGS#2226: ptpSchedulingPriority value updated to 10 (#51637)
openshift/openshift-docs: modules/nw-ptp-configuring-linuxptp-services-as-boundary-clock.adoc

      timeSource                        0xA0
    phc2sysOpts:                        "-a -r -n 24" <10>

>     ptpSchedulingPolicy:                SCHED_OTHER   <11>

Reviewer: There's no reason to specify ptpSchedulingPriority if it's SCHED_OTHER.  Is there a way you can indicate this is optional?

Author: @josephdrichard Hey! Checked callout <12> that is next to `ptpSchedulingPriority` and it says "Integer value from 1-65 used to set FIFO priority for ptp4l and phc2sys processes. *Required if SCHED_FIFO is set for ptpSchedulingPolicy.*"

Would you like me to change the `ptpSchedulingPolicy` to `SCHED_FIFO` so the set `ptpSchedulingPriority` makes sense?

I didn't put the preview links in my first comment, but here it is now: https://docs.openshift.com/container-platform/4.10/networking/using-ptp.html#configuring-linuxptp-services-as-boundary-clock_using-ptp

Reviewer: I don't think that's the right preview link.
In my opinion that should say:
_Required if SCHED_FIFO is set for ptpSchedulingPolicy.  Unused Otherwise._

Author: This should be correct one: https://51637--docspreview.netlify.app/openshift-enterprise/latest/networking/using-ptp.html#configuring-linuxptp-services-as-boundary-clock_using-ptp

Added a comment that the value is unused otherwise.

<9> For Intel Columbiaville 800 Series NICs, ensure `boundary_clock_jbod` is set to `0`. For Intel Fortville X710 Series NICs, ensure `boundary_clock_jbod` is set to `1`.
<10> Specify system config options for the `phc2sys` service. If this field is empty the PTP Operator does not start the `phc2sys` service.
<11> Scheduling policy for ptp4l and phc2sys processes. Default value is `SCHED_OTHER`. Use `SCHED_FIFO` on systems that support FIFO scheduling.

> <12> Integer value from 1-65 used to set FIFO priority for `ptp4l` and `phc2sys` processes. Required if `SCHED_FIFO` is set for `ptpSchedulingPolicy`. This value is unusued otherwise.

Reviewer 2: Suggest clarifying when the field is not used.

```suggestion
<12> Integer value from 1-65 used to set FIFO priority for `ptp4l` and `phc2sys` processes when `ptpSchedulingPolicy` is set to `SCHED_FIFO`. The `ptpSchedulingPriority` field is not used when `ptpSchedulingPolicy` is set to `SCHED_OTHER`.
```

Author: It is explained here: https://51637--docspreview.netlify.app/openshift-enterprise/latest/networking/using-ptp.html#cnf-configuring-fifo-priority-scheduling-for-ptp_using-ptp

I can add it in additional info after each section.
